plugins/network/whois

```diff
@@ -6,35 +6,37 @@
 
 =head1 NAME
 
-whois_ - Plugin to monitor expiry dates of domain in the WHOIS database
+whois - Plugin to monitor expiry dates of domain in the WHOIS database
```

Comment:
Maybe *of domains* (instead of *of domain*)?

Reply:
Right, now fixed.